### PR TITLE
Simplify local image builds

### PR DESCRIPTION
**Pull request by scoopex**

### CODE REVIEW

**Comment by berendt** [comment]
Can you rename `create_image_flavour.sh` to `create-image-flavor.sh`. We have "OpenStack flavors" and I prefer to use `-` in filenames.

**Comment by scoopex** [comment]
> Can you rename `create_image_flavour.sh` to `create-image-flavor.sh`. We have "OpenStack flavors" and I prefer to use `-` in filenames.

Of cou(!)rse :-)

**Comment by berendt** [comment]
@scoopex Please rebase. Looks like this PR included the https://github.com/osism/node-image/commit/8a0fab5d98b1b698b90adf4c743f866564d84e0f.

**Comment by scoopex** [comment]
> @scoopex Please rebase. Looks like this PR included the [8a0fab5](https://github.com/osism/node-image/commit/8a0fab5d98b1b698b90adf4c743f866564d84e0f).

Done :-)

File: create_image_flavour.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (berendt) [inline create_image_flavour.sh:1]: This file should be removed.

File: playbooks/build.yml [hunk 2]
```diff
@@ -12,19 +12,7 @@
...
-          cp variants/runcmd-{{ name }}.yml include.d/runcmd.yml
-
-          pipenv run python3 render-user-data.py > user-data
-          # cloud-init schema --config-file user-data
-
-          bash image-create.sh -r -a -k -u user-data -n jammy
-
-          mv ubuntu-autoinstall.iso ubuntu-autoinstall-{{ name }}.iso
-          sha256sum ubuntu-autoinstall-{{ name }}.iso > ubuntu-autoinstall-{{ name }}.iso.CHECKSUM
+          ./create_image_flavour.sh {{ name }}
```
Comment (berendt) [inline playbooks/build.yml:15]: create-image-flavor.sh

**Review by berendt — CHANGES_REQUESTED** [review]

File: create_image_flavour.sh (hunk 1 above)
Comment (scoopex) [reply create_image_flavour.sh]: Damn and done :-)

File: playbooks/build.yml (hunk 2 above)
Comment (scoopex) [reply playbooks/build.yml]: Damn and done :-)

**Pull request merged**